manifests/04.rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
the file name `04.rolebinding.yaml` has `.` where other files have `_`

Reply:
Fixed